client/lib/domains/resolve-domain-status.js

```diff
@@ -232,7 +232,7 @@ export function resolveDomainStatus(
 			if ( domain.isPendingIcannVerification && domain.currentUserCanManage ) {
 				const noticeText = domain.currentUserIsOwner
 					? translate(
-							'We sent you an email to verify your contact information. Please complete the verification or your domain will stop working. You can also {{a}}change you email address{{/a}} if you like.',
+							'We sent you an email to verify your contact information. Please complete the verification or your domain will stop working. You can also {{a}}change your email address{{/a}} if you like.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.